### PR TITLE
build: Replace `npm` with `pnpm`

### LINKED ISSUE #347: Swap out `npm` for `pnpm`
### Description

Regarding #246, the Tauri team recommends swapping `npm` for `pnpm`.

### Motivation

#246 might be a tool chain bug. Therefore, trying our luck with `pnpm`.

### Resources

_No response_

### To-do List

n/a

### PR DESCRIPTION
**Pull request by maiertech**
# Description of change

Replace `npm` with `pnpm`. Update configs and generate new lock file.

## Links to any relevant issues

- Fixes #347.

## How the change has been tested

- CI tasks should not fail.
- UniMe should start up locally.

## Definition of Done checklist

Add an `x` to the boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@
 
 # UniMe - Identity Wallet
 
-Identity Wallet for people to manage Decentralized Identities and Verifiable Credentials.
+Identity Wallet for people to manage decentralized identities and verifiable credentials.
```
Comment (daniel-mader) [inline README.md]: I believe "**V**erifiable **C**redentials" was intentionally stylized using capital letters.

File: .vscode/settings.json [hunk 2]
```diff
@@ -5,5 +5,9 @@
   },
   "[rust]": {
     "editor.defaultFormatter": "rust-lang.rust-analyzer"
+  },
+
```
Comment (daniel-mader) [inline .vscode/settings.json:9]: Blank line intentional?

**Review by daniel-mader — COMMENTED** [review]
Following [Conventional Commits](https://github.com/angular/angular/blob/68a6a07/CONTRIBUTING.md#commit-message-header), the type of this PR should be a `build:` since it affects the build system.

File: .vscode/settings.json (hunk 2 above)
Comment (maiertech) [reply .vscode/settings.json:9]: I group settings by alphabetic blocks.

File: README.md (hunk 1 above)
Comment (maiertech) [reply README.md]: Changed back.

**Review by daniel-mader — APPROVED** [review]

**Review by Oran-Dan — APPROVED** [review]

**Pull request merged**